ironfish-cli/src/utils/asset.ts

```diff
@@ -34,14 +34,14 @@ export async function selectAsset(
   }
 
   // Get the asset name from the chain DB to populate the display choices
-  for (const { assetId } of balances) {
+  for (const { assetId, confirmed } of balances) {
     const assetResponse = await client.getAsset({ id: assetId })
 
     if (assetResponse.content.name) {
       const displayName = BufferUtils.toHuman(Buffer.from(assetResponse.content.name, 'hex'))
       assetOptions.push({
         value: assetId,
-        name: `${assetId} (${displayName})`,
+        name: `${assetId} (${displayName}) - ${CurrencyUtils.renderIron(confirmed)}`,
```

Comment:
I worry that `$IRON - 1.00000000` might lead some users to think that we're displaying a negative balance here

Reply:
yea, good point. it might just be me, but putting double brackets there feels like bad ux. how about formatting it with`${displayName}, id:${assetId}, balance:${CurrencyUtils.renderIron(confirmed)}`?

<img width="617" alt="Screen Shot 2023-02-01 at 1 35 03 PM" src="https://user-images.githubusercontent.com/2752586/216044484-db9a9891-8d41-464d-b678-8a643ae56a98.png">

<img width="685" alt="Screen Shot 2023-02-01 at 1 36 16 PM" src="https://user-images.githubusercontent.com/2752586/216044476-dba1dcbf-c065-4f3e-b019-53b2d4a082c9.png">

Reply:
in my opinion the second example without brackets feels more intuitive as it is closer to how the output of the balances command.

Reply:
I think I'd like to go with the first one, since it causes the asset IDs to align vertically. I can make that change here quick and merge it